app/index.html

```diff
@@ -308,6 +303,15 @@ <h1 ng-bind="globalPage.title"></h1>
   </script>
   <!-- /build -->
 
-  <script type="text/javascript">_satellite.pageBottom();</script>
+  <!-- Adobe DTM -->
+  <script>
+    window.digitalData = {};
+  </script>
+  <!-- build:remove:dist -->
```

Comment:
True.  Forgot about that.  Take a look at what I committed and let me know what you think.